docs/Contributing/Contributing.md

```diff
@@ -0,0 +1,9 @@
+\cond NEVER
+Distributed under the MIT License.
```

Comment:
Is there a way to disable the animation on the left panel every time I click a link?

Reply:
I haven't figured out yet when it appears. I think it's in https://github.com/doxygen/doxygen/blob/master/templates/html/navtree.js.

Reply:
The call to `docContent.animate` seems suspicious.